project.yaml

```diff
@@ -21,24 +22,32 @@ actions:
 
 # Join data
   join_data:
-    run: r:latest analysis/data_join.R
+    run: python:latest analysis/join_ethnicity.py
     needs: [generate_study_population, generate_study_population_ethnicity]
     outputs:
       highly_sensitive:
-        data: output/input*.csv.gz
+        cohort: output/input*.csv.gz
 
-# Calculate mortality rates
+# Calculate mortality rates (crude + subgroup specific)
   calculate_measures:
     run: cohortextractor:latest generate_measures --study-definition study_definition
     needs: [join_data]
     outputs:
       moderately_sensitive:
-        measure: output/measure_*_rate.csv
+        measure: output/measure_*_mortality_rate.csv
 
-# Standardise mortality rates
+# Standardise crude mortality rate
   standardise_crude_rates:
     run: r:latest analysis/crude_rates_standardise.R
     needs: [calculate_measures]
     outputs:
       moderately_sensitive:
-        measure: output/rates/crude_monthly_std.csv 
\ No newline at end of file
+        rds: output/rates/crude_*monthly_std.rds 
```

Comment:
For viewing on the server, you may ant to save this as csv

Reply:
Good one. Changed now.